README.md

```diff
@@ -1,2 +1,18 @@
 # stacks-java
 Java Web API Application Scaffolding for Amido Stacks
+
+## Run Locally
+
+> ./gradlew bootRun
+
+## Build Docker Image
+
+> docker build -t <image-tag> .
+
+## Swagger/OAS
+
+Automatically generated. Go to: http://localhost:9000/api-docs
```

Comment:
Can we change this endpoint to match what .NET stacks has?

Reply:
.NET is using `/swagger/index.html` - though not sure I like it. It's easy to reconfigure the path though

Reply:
done